.vscode/settings.json

```diff
@@ -55,4 +55,5 @@
     "esbonio.sphinx.confDir": "${workspaceFolder}/source",
     "files.trimTrailingWhitespace": true,
     "rewrap.wrappingColumn": 79,
+    "makefile.configureOnOpen": false,
```

Comment:
Was this an intentional change?

Reply:
This came from me, somehow.  But yes.  This keeps VS code from attempting to run `./configure` for Makefile projects.